api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java

```diff
@@ -240,23 +240,23 @@ public void purgeConcept(Concept concept) throws DAOException {
 	 */
 	@Override
 	public Concept getConcept(Integer conceptId) throws DAOException {
-		return (Concept) sessionFactory.getCurrentSession().get(Concept.class, conceptId);
+		return sessionFactory.getCurrentSession().get(Concept.class, conceptId);
 	}
 	
 	/**
 	 * @see org.openmrs.api.db.ConceptDAO#getConceptName(java.lang.Integer)
 	 */
 	@Override
 	public ConceptName getConceptName(Integer conceptNameId) throws DAOException {
-		return (ConceptName) sessionFactory.getCurrentSession().get(ConceptName.class, conceptNameId);
+		return sessionFactory.getCurrentSession().get(ConceptName.class, conceptNameId);
 	}
 	
 	/**
 	 * @see org.openmrs.api.db.ConceptDAO#getConceptAnswer(java.lang.Integer)
 	 */
 	@Override
 	public ConceptAnswer getConceptAnswer(Integer conceptAnswerId) throws DAOException {
-		return (ConceptAnswer) sessionFactory.getCurrentSession().get(ConceptAnswer.class, conceptAnswerId);
```

Comment:
Are these accidental changes?

Reply:
@dkayiwa  As if line 259 in my file differs. https://snipboard.io/8Bm1lw.jpg

Reply:
That is exactly why am asking. Did you accidentally change it from what it originally was? See the line in red above.

Reply:
@dkayiwa i have managed to rectify the problem